src/Data/Units/PartsPer.purs

```diff
@@ -0,0 +1,26 @@
+-- | A collection of parts-per-x units.
+module Data.Units.PartsPer where
+
+import Prelude
+
+import Data.Units (DerivedUnit, makeNonStandard, unity)
+
+-- | 'Dimensionless' ratio *1 pct = 1e-2*.
+pct ∷ DerivedUnit
```

Comment:
I think we should use long names for the units here, i.e. `percent` and `partsPerMillion`.

Reply:
Sure thing. Updated.